packages/react-instantsearch/src/connectors/connectMenu.js

```diff
@@ -87,11 +99,16 @@ export default createConnector({
       console.warn('react-instantsearch: `searchForFacetValues` has been renamed to' +
         '`withSearchBox`, this will break in the next major version.');
     }
+    if (props.withSearchBox && this.context.multiIndexContext) {
+      // eslint-disable-next-line no-console
+      console.error('react-instantsearch: searching in *List is not available when used inside a' +
```

Comment:
Can you also add a comment for reference to understand why it's not available. And what would be needed to make it working.